### PR TITLE
Self-reference heimdall using the master branch instead of 3.0.0

### PR DESCRIPTION
**Pull request by vthiery**
The previous version referenced (3.0.0) was not enough to support the examples. I self-referenced the package with the branch `master` to always be synced

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -41,6 +41,10 @@
   name = "github.com/stretchr/testify"
   version = "1.1.4"
 
+[[constraint]]
```
Comment (rShetty) [inline Gopkg.toml]: Sorry I did not get why is this required ? Is this for examples section ?

Comment (vthiery) [reply Gopkg.toml]: Yes I had issues running the examples without pinning the version of Heimdall to a "recent" SHA. I just tried to reproduce my issue but it all works fine so we can close this PR.
Thanks!

Comment (rShetty) [reply Gopkg.toml]: 👍
